docs/commands/join.md

```diff
@@ -1,59 +1,68 @@
+Apart from providing individual API definition files as the input, you can also specify the path to a folder that contains multiple API definition files and match them with a wildcard (for example, `myproject/openapi/*.yaml`). The `join` command will collect all matching files and combine them into one file.
 
-Only OpenAPI 3.x definitions are supported by this command.
-
-
-### `join` usage
+### Usage
 
+```bash
+openapi join <entrypoint> <entrypoint>...
+openapi join <path-to-folder>/<wildcard-pattern> [--lint]
+openapi join [--help] [--prefix-components-with-info-prop] [--prefix-tags-with-info-prop] [--prefix-tags-with-filename]
 
-```
-Positionals:
-entrypoints                                              [array] [default: []]
-
-Options:
---version                           Show version number.             [boolean]
---help                              Show help.                       [boolean]
---lint                              Lint definitions[boolean] [default: false]
---prefix-tags-with-info-prop        Prefix tags with property value from info
-object                            [string]
---prefix-tags-with-filename         Prefix tags with property value from file
-name            [boolean] [default: false]
---prefix-components-with-info-prop  Prefix components with property value from
-info object                       [string]
+openapi join first-api.yaml second-api.yaml
+openapi join first-api.yaml second-api.json
+openapi join ./*.yaml
+openapi join --version
 ```
 
+## Options
 
-The command:
+option                               | type               | required?    | default     | description
+-------------------------------------|:------------------:|:------------:|:-----------:|------------
+`entrypoints`                        | `array`, wildcard  | yes          | `[]`, -     | 1. Array of paths to API definition files that need to be joined. At least two input files are required.<br />2. A wildcard pattern to match API definition files within a specific folder
+`--help`                             | `boolean`          | no           | -           | Show help
+`--lint`                             | `boolean`          | no           | `false`     | Lint definition files
+`--prefix-components-with-info-prop` | `string`           | no           | -           | Prefix components with property value from info object. See the [section](#) below
+`--prefix-tags-with-info-prop`       | `string`           | no           | -           | Prefix tags with property value from info object. See the [section](#) below
+`--prefix-tags-with-filename`        | `boolean`          | no           | `false`     | Prefix tags with property value from file name. See the [section](#) below
```

Comment:
These links don't lead anywhere - what should they link to? Also, please try using a more descriptive anchor text - if all sentences just say "See the section below", it could be difficult for the readers to distinguish which section you're talking about.